engines/python/setup/djl_python/deepspeed.py

```diff
@@ -357,6 +357,10 @@ def create_model_pipeline(self):
             self.ds_config["replace_with_kernel_inject"] = False
         state_dict_mmap = {}
         if "checkpoint" in self.ds_config:
+            if self.quantize_mode:
+                raise ValueError(
+                    f"Quantization does NOT currently work WITH checkpoint loading. "
+                    f"Please turnoff quantization and try again.")
```

Comment:
1. typo: "turnoff" -> "turn off"
2. Let's say "`quantize_mode` option does NOT currently work WITH DeepSpeed checkpoints using `checkpoint` option." to be painfully clear
3. Could first suggest using quantization with a standard HuggingFace checkpoint, before adding "or turn off quantization". Our new model loading with a standard checkpoint on SageMaker is even more memory efficient and almost as fast to load as a presharded DeepSpeed checkpoint.

Reply:
Done